src/app/components/login-page/login-page.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { LoginService } from '../../services/login-service/login-service.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-login-page',
+  templateUrl: './login-page.component.html',
+  styleUrls: ['./login-page.component.scss']
+})
+export class LoginPageComponent implements OnInit {
+
+  usernameInput: string;
+  passwordInput: string;
+  loginMessage: string;
+
+  constructor(private loginService: LoginService, public router: Router) { }
+
+  sendLogin() {
+    if (this.usernameInput && this.passwordInput) {
+      this.loginService.login(this.usernameInput, this.passwordInput).subscribe((response) => {
+        if (response.status === 200) {
+          this.router.navigate([`/heroes-hall/${response.body.id}`]);
```

Comment:
The response shouldn't contain the entire response body. Only pass the necessary information, so the service will function by proper programming principles.

Reply:
Fixed thank you.